src/index.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
 import './index.css';
+import store from './redux/configureStore';
 
 ReactDOM.render(
   <React.StrictMode>
-    <Router>
-      <App />
-    </Router>
+    <Provider store={store}>
+      <Router>
+        <App />
+      </Router>
```

Comment:
- [ ] You have used the routes in the app.js file, no need to wrap the app here again.

Reply:
I'm sorry but, it doesn't work if I take it out. I tried.

Reply:
You can leave it now but try to find out the way it should be solved.